keys.go

```diff
@@ -224,8 +239,8 @@ func (c *Client) GetKeyMetadata(ctx context.Context, id string) (*Key, error) {
 }
 
 //ListKeyVersions retrieves all the rotated key versions associated with the key
-func (c *Client) GetKeyVersions(ctx context.Context, id string) (*Key, error) {
-	keys := Keys{}
+func (c *Client) ListKeyVersions(ctx context.Context, id string) (*KeyVersions, error) {
```

Comment:
You'll want to return a []KeyVersion here.

Reply:
the output matches the  keyVersions struct. So I believe keyVersions should be fine